main.lua

```diff
@@ -41,7 +41,13 @@ expose("ptusa", function()
 
                 obj:set_mode( i, 1 )
                 assert.are.equal( ptusa.eval(), 0 )
-
+                local m_mngr = obj:get_modes_manager()
+                local oper = m_mngr[ i ]
+                local count_steps = oper[ operation.RUN ]:steps_count()
+                for j = 1, count_steps do
+                    print( "\t"..j )
+                    oper:to_step( j )
```

Comment:
Предлагаю также добавить выполнение операции после перехода к шагу:
```suggestion
                    oper:to_step( j )
                    assert.are.equal( ptusa.eval(), 0 )
```

Reply:
Как скажете